jobfunnel/indeed.py

```diff
@@ -187,43 +191,47 @@ def scrape(self):
                 job['date'] = ''
 
             try:
-                # Added capture group so to only capture id once matched.
+                # added capture group so to only capture id once matched
+                id_regex = re.compile(r'id=\"sj_([a-zA-Z0-9]*)\"')
```

Comment:
You should move the id_regex out of the for loop as it only needs to be compiled once, instead of re-compiling it for every job we have. [Reference](https://docs.python.org/3/library/re.html#re.compile).

Reply:
You're right.  I moved it out of the way to make the provider files look more consistent but it is now performing redundant tasks.  Thank you for noticing.  I also noticed that I removed your replacement of '+' in the query string to '-'.  I will change this back.

Reply:
Perfect, I'll go ahead and approve now so you can merge right away after uploading the commit.